### ISSUE TITLE
random comment

### ISSUE THREAD

**Issue by MichelNivard** [sashagusev/tan2024_cog_hsq#1]
I'd run all traits, these are potentially just magnitude changes that are homogeneous across all traits?

**Comment by sashagusev** [comment]
Good idea. I ran many but not all of the Tan et al. traits and did not see changes in the magnitude nearly as strong as for CP. I think there might be something genuinely different about the genetic architecture due to cohort selection or direct/indirect effect relationships.
